test/e2e/assets-client-fetch/fixture/src/home.js

```diff
@@ -1,4 +1,4 @@
-// @flow
+// @noflow
```

Comment:
Why noflow? Can we $FlowFixMe instead? (Same for other noflow changes in next few files)